### PR TITLE
Disable kAutofillWalletImportEnabled which was enabled by default

### PR DESCRIPTION
**Pull request by darkdh**
and hide its related sync toggle

Resolves https://github.com/brave/brave-browser/issues/10453

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [ ] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Test Plan:
1. Create sync chain
2. After creation and go to brave://settings/braveSync/setup
3. This toggle should be hidden
Screen Shot 2020-06-24 at 13 04 18

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

**Comment by darkdh** [comment]
skip CI for now, will do rebase when cr84 merged into master

File: browser/brave_profile_prefs.cc [hunk 1]
```diff
@@ -24,6 +24,7 @@
 #include "components/content_settings/core/common/pref_names.h"
 #include "components/embedder_support/pref_names.h"
 #include "components/gcm_driver/gcm_buildflags.h"
+#include "components/autofill/core/common/autofill_prefs.h"
```
Comment (bridiver) [inline browser/brave_profile_prefs.cc:27]: do we already have a dep for this?

Comment (darkdh) [reply browser/brave_profile_prefs.cc:27]: yes, already has the dep

File: browser/resources/settings/brave_overrides/sync_controls.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import {html} from 'chrome://resources/polymer/v3_0/polymer/polymer_bundled.min.js'
```
Comment (petemill) [inline browser/resources/settings/brave_overrides/sync_controls.js:6]: This import is not used, can be removed

Comment (darkdh) [reply browser/resources/settings/brave_overrides/sync_controls.js]: fixed, thanks.

**Comment by darkdh** [comment]
Unrelated error on macos
```
22:47:20  ../../brave/browser/extensions/brave_component_loader.h:12:10: fatal error: 'brave/components/brave_rewards/browser/buildflags/buildflags.h' file not found
```
https://ci.brave.com/job/pr-brave-browser-disable-sync-payment-integration/4/execution/node/314/log/
rerun macos only

**Comment by darkdh** [comment]
known issue for `audit-deps`
https://ci.brave.com/job/pr-brave-browser-disable-sync-payment-integration/5/execution/node/325/log/
and
```
08:27:50  [148/149] ACTION //brave/test:fix_test_install_name_brave_browser_tests(//build/toolchain/mac:clang_x64)
08:27:50  FAILED: alwaysrunbrowsertests/fix_test_install_name_brave 
08:27:50  python ../../build/gn_run_binary.py /usr/bin/install_name_tool -change @loader_path/../../Libraries/libadblock.dylib @executable_path/gen/adblock/out/x86_64-apple-darwin/release/libadblock.dylib -change @loader_path/../../Libraries/libchallenge_bypass_ristretto.dylib @executable_path/gen/challenge_bypass_ristretto/out/x86_64-apple-darwin/release/libchallenge_bypass_ristretto.dylib -change @loader_path/../../Libraries/libspeedreader_ffi.dylib @executable_path/gen/speedreader/out/x86_64-apple-darwin/release/libspeedreader_ffi.dylib brave_browser_tests
08:27:50  error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/install_name_tool: changing install names or rpaths can't be redone for: brave_browser_tests (for architecture x86_64) because larger updated load commands do not fit (the program must be relinked, and you may need to use -headerpad or -headerpad_max_install_names)
08:27:50  /usr/bin/install_name_tool failed with exit code 1
```
https://ci.brave.com/job/pr-brave-browser-disable-sync-payment-integration/5/execution/node/382/log/

**Pull request merged**